src/utils/links.ts

```diff
@@ -15,16 +15,22 @@ export function openGitifyReleaseNotes(version: string) {
   );
 }
 
-export function openGitHubNotifications() {
-  openExternalLink('https://github.com/notifications' as Link);
+export function openGitHubNotifications(account: Account) {
+  const url = new URL(`https://${account.hostname}`);
+  url.pathname = 'notifications';
+  openExternalLink(url.toString() as Link);
```

Comment:
As I can see all use cases of `openExternalLink` pass string as a parameter. So I would say that probably it should accept the argument as `string` since all external URLs are not created as a branded `Link`